servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ProtocolConfigBuilder.java

```diff
@@ -103,33 +106,79 @@ public H2ProtocolConfigBuilder keepAlivePolicy(final KeepAlivePolicy policy) {
         return this;
     }
 
+    /**
+     * Sets the initial <a href="https://datatracker.ietf.org/doc/html/rfc7540#section-6.5.1">HTTP/2 Setting</a> to for
+     * each h2 connection.
+     * @param settings the initial settings to for each h2 connection.
+     * @return {@code this}
+     * @see Http2SettingsBuilder
+     */
+    public H2ProtocolConfigBuilder initialSettings(Http2Settings settings) {
+        this.h2Settings = requireNonNull(settings);
+        return this;
+    }
+
+    /**
+     * Provide a hint on the number of bytes that the flow controller will attempt to give to a stream for each
+     * allocation (assuming the stream has this much eligible data).
+     * @param flowControlQuantum a hint on the number of bytes that the flow controller will attempt to give to a
+     * stream for each allocation (assuming the stream has this much eligible data).
+     * @return {@code this}
+     */
+    public H2ProtocolConfigBuilder flowControlQuantum(int flowControlQuantum) {
+        if (flowControlQuantum <= 0) {
+            throw new IllegalArgumentException("flowControlQuantum " + flowControlQuantum + " (expected >0)");
+        }
+        this.flowControlQuantum = flowControlQuantum;
+        return this;
+    }
+
     /**
      * Builds {@link H2ProtocolConfig}.
      *
      * @return {@link H2ProtocolConfig}
      */
     public H2ProtocolConfig build() {
-        return new DefaultH2ProtocolConfig(headersFactory, headersSensitivityDetector, frameLoggerConfig,
-                keepAlivePolicy);
+        return new DefaultH2ProtocolConfig(h2Settings, headersFactory, headersSensitivityDetector, frameLoggerConfig,
+                keepAlivePolicy, flowControlQuantum);
     }
 
-    private static final class DefaultH2ProtocolConfig implements H2ProtocolConfig {
+    static Http2SettingsBuilder newDefaultSettingsBuilder() {
+        return new Http2SettingsBuilder()
+                .initialWindowSize(1048576) // 1mb default window size
+                .maxHeaderListSize((int) DEFAULT_HEADER_LIST_SIZE);
+    }
+
+    /**
+     * Default allocation quantum to use for the remote flow controller.
+     * @return Default allocation quantum to use for the remote flow controller.
+     */
+    private static int defaultFlowControlQuantum() {
+        return 1024 * 16;
```

Comment:
Since this method is private, consider assigning the value directly to the `flowControlQuantum`, then we can kill the method. It will be easier for developers to read default value without the need to jump to the end of the class.

Reply:
I killed the method.